gulp_tasks/locales.js

```diff
@@ -1,12 +1,50 @@
 const gulp = require('gulp');
-const rename = require('gulp-rename');
+const fs = require('fs');
+const path = require('path');
+const merge = require('merge-stream');
+const merge_json = require('gulp-merge-json');
 
-gulp.task('locales', locales);
+const conf = require('../conf/gulp.conf');
+const localesPath = 'bower_components/impac-angular/dist/locales/';
+
+// Return a list of locales file from a folder
+function getLocales(dir) {
+  return fs.readdirSync(dir)
```

Comment:
Can't you do something like `return fs.readdirSync(path.join(dir + "/*.json")`

Reply:
Nope, but I've solved it using `glob.sync`.